src/components/MessageDisplayer.tsx

```diff
@@ -85,7 +85,7 @@ export class MessageDisplayerComponent extends React.Component<IPorps, {}> {
             <div className={classes.chatroom}>
                 <ul className={classes.chats}>
                     
-                    {messageStore !== undefined && messageStore.getMessagesFromAddress.length > 0 ? messageStore.getMessagesFromAddress.map(
+                    {messageStore !== undefined && messageStore.messages.length > 0 ? messageStore.messages.map(
```

Comment:
you will always get all messages from all chats and not only the messages from your selected chat.